biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -2490,8 +2511,8 @@ subroutine hydraulics_bc ( nsites, sites, bc_in, bc_out, dtime)
              call UpdatePlantPsiFTCFromTheta(ccohort,site_hydr)
 
              ccohort_hydr%btran = wkf_plant(stomata_p_media,ft)%p%ftc_from_psi(ccohort_hydr%psi_ag(1))
-             
-
+             ! write(fates_log(),*) 'cohort(dbh):', ccohort%dbh, 'btran', ccohort_hydr%btran            
```

Comment:
please remove commented out write statements before we get to final integration phase